aspnet/dnx/commands.rst

```diff
@@ -1,8 +1,121 @@
-.. include:: /../common/stub-topic.txt
+Using Commands
+==============
 
-|stub-icon| Using Commands
-==========================
+DNX projects are used to build and run .NET applications for Windows, Mac and Linux. DNX provides a host process, CLR hosting logic and managed entry point discovery. You can use DNX to execute commands from a command prompt. 
 
-.. include:: /../common/stub-notice.txt
-    
-.. _issue: https://github.com/aspnet/Docs/issues/55
+What is a command?
+------------------
+
+A command is a named execution of a .NET entry point with specific arguments. The *project.json* file allows you to define commands for your projects. Those commands are understood by Visual Studio Code (VS Code) and they will show up in VS Code's **Command Palette**.
```

Comment:
> A command is a named execution of a .NET entry point with specific arguments. The *project.json* file allows you to define commands for your projects.

A command is a named execution of a .NET entry point with specific arguments. Commands can be defined locally in your project or installed globally on your machine. The *project.json* file allows you to define commands for your projects and define commands that you want to make available globally.